README.md

```diff
@@ -1,2 +1,112 @@
-# aue
-Calculations for best- and worst-case scenario adult-use establishment distributions
+# Adult Use Establishments Proximity Analysis
+
+This repo contains code for identifying the best- and worst-case scenarios for the spatial distribution of Adult Use Establishments in NYC, given the restriction that no establishment be within 500ft of another establishment.
+
+## Background
+
+## Usage
+
+Run `./aue.sh` from the home directory. Results are found in the `output` directory. Each row contains the BBLs for a single solution. The number of columns is the total number of possible establishments. These counts are also output to the terminal.
+
+## Overview of Results
+
+Using the methodology outlined below, we have identified a best-case scenario of **208** lots that could feasibly have Adult Use Establishments, given the distance restrictions. The worst-case scenario, meaning the case where establishments are arranged in a way that limits the addition of new establishments, is **203** lots. Note that these numbers are the maximum *possible* establishments. In reality, not all lots are currently vaccant.
+
+These numbers are dependent on the input data, however, and there is ongoing work vetting the input lots as realistic locations for an establishment (i.e. there is sufficient street-frontage and access). In the future, this workflow can be re-run with refined input data to improve the estimate of best- and worst-case scenarios, simply by modifying the path for the input buffered lots.
+
+# Methodology
+
+We approached this problem by creating a matrix of possible co-existing establishments. Simplifying the distance calculations to a binary flag makes our calculations much easier. 
```

Comment:
I think you need a little more substance and clarity here.  I think that you might lose an audience with this description.  

Also, the problem isn't clearly stated above this.
The problem: using a subset of lots that meet a criteria what is the maximum number of establishments you can site given that each establishment needs to be at least 500 feet away from another establishments.